docs/sitegen.yaml

```diff
@@ -52,26 +52,20 @@ backend:
         type: "image"
         value: "images/helidon_logo_dark.svg"
       items:
+        - type: "PAGE"
+          title: "Introduction"
+          source: "about/introduction.adoc"
+          glyph:
+            type: "icon"
+            value: "widgets"
+        - type: "PAGE"
+          title: "Helidon CLI"
+          source: "about/cli.adoc"
+          glyph:
+            type: "icon"
+            value: "widgets"
         - type: "GROUPS"
           items:
-          - type: "GROUP"
-            title: "Introduction"
-            dir: "about"
-            items:
-              - type: "MENU"
-                title: "About"
-                glyph:
-                  type: "icon"
-                  value: "assistant"
-                sources:
-                  - "overview.adoc"
-                  - "introduction.adoc"
-                  - "prerequisites.adoc"
-                  - "managing-dependencies.adoc"
-                  - "windows.adoc"
-                  - "cli.adoc"
-                  - "kubernetes.adoc"
-                  - "upgrade.adoc"
```

Comment:
Where is our upgrade doc going to be located?
Also I am missing SE introduction page (this change only contains MP)

Reply:
Mitia is working on migration guides. I expect adding links to the corresponding introduction pages as part of his commit.

Reply:
This PR doesn't cover SE introduction page. IMO, it doesn't need modifications.